templates/records/record_disambiguation_page.html

```diff
@@ -48,8 +48,8 @@ <h2 class="sr-only">Associated records</h2>
                 {% endif %}
                 <li class="pagination__list-pages" aria-label="Page numbers">
                     <ol>
-                        {% for i in pages.paginator.page_range %}
-                            {% if pages.number == i %}
+                        {% for i in record_pages.paginator.page_range %}
```

Comment:
Really, there's no reason why the `paginator` itself shouldn't be added to the context, then this could become

```suggestion
                        {% for i in paginator.page_range %}
```

Reply:
Done to reflect `record_results_page`